deisctl/units/deis-controller.service

```diff
@@ -9,7 +9,9 @@ EnvironmentFile=/etc/environment
 TimeoutStartSec=20m
 ExecStartPre=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/controller` && docker history $IMAGE >/dev/null 2>&1 || docker pull $IMAGE"
 ExecStartPre=/bin/sh -c "docker inspect deis-controller >/dev/null 2>&1 && docker rm -f deis-controller || true"
-ExecStart=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/controller` && docker run --name deis-controller --rm -p 8000:8000 -e EXTERNAL_PORT=8000 -e HOST=$COREOS_PRIVATE_IPV4 -v /var/run/fleet.sock:/var/run/fleet.sock -v /var/lib/deis/store:/data $IMAGE"
+ExecStart=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/controller` && docker run --name deis-controller --rm -p 8000:8000 -e EXTERNAL_PORT=8000 -e HOST=$COREOS_PRIVATE_IPV4 -e ETCD=$COREOS_PRIVATE_IPV4:4001 -v /var/run/fleet.sock:/var/run/fleet.sock -v /var/lib/deis/store:/data $IMAGE"
+ExecStartPost=/bin/sh -c "echo 'Waiting for controller on 8000/tcp...' && until ncat $COREOS_PRIVATE_IPV4 8000 --exec '/usr/bin/echo dummy-value' >/dev/null 2>&1; do sleep 1; done"
+ExecStartPost=/usr/bin/docker exec deis-controller ./manage.py load_db_state_to_etcd
```

Comment:
I think calling the command [here](https://github.com/deis/deis/blob/master/controller/bin/boot#L88) instead would be more consistent.